core/server/services/stats/lib/mrr-stats-service.js

```diff
@@ -0,0 +1,154 @@
+const {DateTime} = require('luxon');
```

Comment:
Luxon was already in the package.json and the lock file, the timezone is set to UTC somewhere too. So I figured it would be better to already start using Luxon in new services too (which will probably also move to a different repository in the future). Should I change it to moment?

Reply:
Let's stick with moment for now and migrate everything in one go.